omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -347,6 +347,7 @@
 # the /_status endpoint.
 
 default['private_chef']['opscode-erchef']['health_ping_timeout'] = 400
+default['private_chef']['opscode-erchef']['include_version_in_status'] = true
```

Comment:
Let us leave the default value to be false. And customers can opt in as per their security preferences.

Reply:
ok sure